spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/File.scala

```diff
@@ -209,6 +211,30 @@ object File {
       if (null != fs) fs.close()
     }
   }
+  def downloadFromUrl(url: String, dir: String): String = {
```

Comment:
please add java doc to the public API

Reply:
Please take a look at FileReader, I think you should enhance that class to make it support http